ci_scripts/generate_docs.py

```diff
@@ -0,0 +1,30 @@
+"""Script to generate documentation using pdoc"""
+import logging
+from subprocess import check_call
+
+from utils.definitions import MODULES_TO_DOCUMENT, DOCUMENTATION_DIR_PATH
+
+logger = logging.getLogger(__name__)
+
+
+def main() -> None:
+    """Triggers building the documentation.
+
+    Modules to document and the output destination path
+    can be set in the utils.definitions config file.
+    """
+    command_list = [
```

Comment:
could the code be surrended by a `ty: except ` similar to this: https://github.com/ARMmbed/mbed-targets/blob/34fdfe05d6deed2cb26f243e597aa510a0586fc9/ci_scripts/publish_target_validation_report.py#L36?

(This was introduced because of Graham's point that if an exception is raised, we may not want the traceback to be shown)

Reply:
Cool, I will wrap the `check_call` 👍